test/invoice_item_test.rb

```diff
@@ -3,22 +3,34 @@
 require_relative '../lib/invoice_item'
 
 class InvoiceItemsTest < MiniTest::Test
+  attr_reader :parent, :invoice_item
+
+  def setup
+    sales_engine = SalesEngine.new
+    @parent = InvoiceItemRepository.new('test/support/sample_invoice_items.csv', sales_engine)
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.